gravitee-apim-console-webui/src/management/api/documentation-v4/components/documentation-empty-state/api-documentation-v4-empty-state.component.html

```diff
@@ -17,25 +17,12 @@
 -->
 <div class="empty">
   <div class="empty__icon"><mat-icon svgIcon="gio:search"></mat-icon></div>
-  <div class="mat-h2">No pages available yet</div>
-  <div class="mat-body-2" *gioPermission="{ anyOf: ['api-documentation-c'] }">Start creating pages to fill up your folder.</div>
-  <div class="">
-    <button
-      class="empty__action"
-      mat-flat-button
-      color="primary"
-      *gioPermission="{ anyOf: ['api-documentation-c'] }"
-      [matMenuTriggerFor]="menu"
-      [disabled]="isReadOnly"
-    >
-      Add new page
-    </button>
-
-    <mat-menu #menu="matMenu">
-      <button *ngFor="let pageType of pageTypes" (click)="onAddPage.emit(pageType)" mat-menu-item>
-        {{ getTitleForPageType(pageType) }}
-        <img [ngSrc]="getLogoForPageType(pageType)" height="24" width="24" [alt]="getTitleForPageType(pageType).toLowerCase() + ' logo'" />
-      </button>
-    </mat-menu>
-  </div>
+  <div class="mat-h2">{{ emptyPageTitle }}</div>
+  <div class="mat-body-2" *gioPermission="{ anyOf: ['api-documentation-c'] }">{{ emptyPageMessage }}</div>
```

Comment:
It would make sense for this `*gioPermission` to be on the button rather than the page message.

Reply:
Done